### PR TITLE
Add See more functionality 

### PR DESCRIPTION
**Pull request by MassiiNechiche**
## Changes proposed
- Concised the list of users showed in home page
- Add a see more button to show more users 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1093"></a>

### CODE REVIEW

File: public/data/massii-nechiche.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "icon": "github"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started full stack job",
+      "date": "october 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started my full stack developer journey at dnsolutions-it"
```
Comment (KendallDoesCoding) [inline public/data/massii-nechiche.json:19]: Add full stop in the end of this line.

File: public/data/massii-nechiche.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/massiinechiche",
+      "icon": "github"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started full stack job",
+      "date": "october 2021",
```
Comment (KendallDoesCoding) [inline public/data/massii-nechiche.json:16]: Make october upper-case

File: public/data/massii-nechiche.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Massii nechiche",
```
Comment (KendallDoesCoding) [inline public/data/massii-nechiche.json:2]: Make, first character of surname "N" upper-case

File: public/data/massii-nechiche.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/massiinechiche",
+      "icon": "github"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started full stack job",
```
Comment (KendallDoesCoding) [inline public/data/massii-nechiche.json:15]: Maybe, it can be full-stack job? But, for now it's fine as it is.

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

**Comment by KendallDoesCoding** [comment]
@MassiiNechiche Please mention in title or description, you added your profile too. It should be in another PR ideally, but you merged it with your other PR so that's fine too.

**Comment by MassiiNechiche** [comment]
hello, i fixed according to your comments and made a new commit

**Comment by emmalearnscode** [comment]
Hi @MassiiNechiche thanks for contributing. Please remove your profile and add it in a different PR. I think this would be good to discuss in an issue before adding to the project in order to discuss the pros and cons of the changes.

**Review by KendallDoesCoding — APPROVED** [review]

**Comment by MassiiNechiche** [comment]
> Hi @MassiiNechiche thanks for contributing. Please remove your profile and add it in a different PR. I think this would be good to discuss in an issue before adding to the project in order to discuss the pros and cons of the changes.

hello, just removed my profile from the commits... i will do add it in another PR, but should i close this PR as well?

**Review by AlphaVS-76 — APPROVED** [review]

File: src/Components/Home/Users.js [hunk 5]
```diff
@@ -74,7 +74,7 @@ function Users({ list }) {
       <div className="user-list flex flex-wrap justify-content-center">
         {!!filteredList &&
           filteredList.length > 0 &&
-          filteredList.map((user, key) => (
+          filteredList.slice(0, 25).map((user, key) => (
```
Comment (KarlGusta) [inline src/Components/Home/Users.js]: What if you sliced it at 24? It looks nice that way. The see more functionality works well.

Comment (MassiiNechiche) [reply src/Components/Home/Users.js]: Feel free to change it as you please sir,
;)

**Review by krshdev — COMMENTED** [review]
Just linking this PR, because a infinity scroll is in progress and I don't think see more and infinity scroll fit well together 👀 #1201

**Comment by eddiejaoude** [comment]
Also note as we are hitting the rate limit for the homepage, we are looking to revamp it, more info here https://github.com/EddieHubCommunity/LinkFree/issues/1264

**Comment by emmalearnscode** [comment]
Thanks so much for contributing this pull request. The code looks good, however, we've decided to take the project in a different direction and solved the issue in a different way and therefore this change is no longer needed.

Please feel free to help out with other issues in the future.

**Pull request closed without merging**